### PR TITLE
2022-07-19 Release

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

**Review by shawntanzk — COMMENTED** [review]
is it an issue that ro.obo doesnt seem to have the OMO terms changes. ro-base.obo seems to have this though, so it feels a bit strange

**Comment by anitacaron** [comment]
I can see the OMO term in the ro.owl. Maybe something is wrong with obo conversion?

**Comment by shawntanzk** [comment]
it seems specific to ro.obo cause ro-base.obo has the axioms so its the specific conversion

**Comment by anitacaron** [comment]
I don't see the OMO addition in ro-base.obo

**Review by matentzn — COMMENTED** [review]
https://github.com/oborel/obo-relations/blob/master/src/ontology/Makefile#L44

Base wont have any OMO stuff, but ro.obo should. weird.

**Comment by shawntanzk** [comment]
> Base wont have any OMO stuff, but ro.obo should. weird.

ironically ro-base.owl has it, but not ro.obo >.< lol

File: src/ontology/subsets/ro-chado.obo.tmp [hunk 1]
```diff
@@ -24,7 +24,7 @@ property_value: http://purl.org/dc/elements/1.1/description "The OBO Relations O
 property_value: http://purl.org/dc/elements/1.1/title "OBO Relations Ontology" xsd:string
 property_value: http://purl.org/dc/terms/license https://creativecommons.org/publicdomain/zero/1.0/
 property_value: http://xmlns.com/foaf/0.1/homepage " https://github.com/oborel/obo-relations/" xsd:anyURI
-property_value: owl:versionInfo "2022-05-23" xsd:string
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply src/ontology/subsets/ro-chado.obo.tmp:27]: I fixed this in the ODK migration.

**Comment by anitacaron** [comment]
The OMO is in the base because it's inside edit and not via an import, as you said here, @matentzn: https://github.com/oborel/obo-relations/pull/615#pullrequestreview-1035504762

The problem is with OBO conversion.

**Comment by anitacaron** [comment]
I can create the OMO import and rerun the release.

**Comment by anitacaron** [comment]
After meeting with @matentzn, we're aware that OBO format doesn't show the new annotation and consequently the terms that are using the annotation. Please avoid using OBO artefact.

**Comment by shawntanzk** [comment]
> After meeting with @matentzn, we're aware that OBO format doesn't show the new annotation and consequently the terms that are using the annotation. Please avoid using OBO artefact.

Is the plan then just to add that in release notes?

**Comment by shawntanzk** [comment]
also little bit confused about the OMO import - is this just jim not putting the new OMO terms in OMO yet and the import is plans for when it moves over?

**Comment by matentzn** [comment]
I think the OBO artefact has been dropping a lot of stuff from RO anyways - it was always highly incomplete. It should just not be used at all, ideally we remove it, but I am not aware of anyone using it right now. We tried to understand why our specific expand annotation was removed (somehow a large portion of RO is in the owl-axioms header), but gave up. I don't think we need to add this to the release notes - this is not new at all, it was always like this.

OMO import is a placeholder for when we do the right thing with ODK.

**Review by shawntanzk — APPROVED** [review]

**Comment by cmungall** [comment]
Can you make a separate issue for me to look at the obo format, and of
course the obo format will not be dropped!

**Comment by cmungall** [comment]
It looks like there is a separate issue with OMO that is totally unrelated to obo format - let's fix this first:

 - #619

**Pull request merged**